packages/plugin-char/src/CharFormatHtmlDomRenderer.ts

```diff
@@ -21,9 +21,9 @@ export class CharFormatHtmlDomRenderer extends FormatDomRenderer {
         // attributes.
         let rendering: Node[];
         const attributes = node.modifiers.find(Attributes);
-        if (attributes && attributes.length) {
+        if (attributes?.length) {
```

Comment:
Well technically , this would be better but it's nitpicking at this point:
```typescript
if (node.modifiers.find(Attributes)) {
```

Reply:
I _have_ to test the length as well because `attributes` might exist but contain no attributes. If that's the case then I should absolutely not wrap the text node in a span. Then all char nodes would have been effectively wrapped in spans in our demo. I don't think that's what you want :-D